overlay/etc/dnsmasq.d/steamcache.conf

```diff
@@ -0,0 +1,55 @@
+# Run as root to make things easy.
+user=root
+group=root
+
+# Add Google's public DNS servers as a fallback.
```

Comment:
We should avoid hard coding configuration such as upstream name servers as there are environments where our values won't work.